recipes-scanners/clamav/clamav_1.4.1.bb

```diff
@@ -0,0 +1,226 @@
+SUMMARY = "ClamAV anti-virus utility for Unix - command-line interface"
+DESCRIPTION = "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats."
+HOMEPAGE = "http://www.clamav.net/index.html"
+SECTION = "security"
+LICENSE = "LGPL-2.1-only"
```

Comment:
Also it's annoying, but the `LICENSE` and `LIC_FILES_CHKSUM` variables should list all possible licenses that apply to the recipe source code. Which means you need to go through the licenses in [`clamav:COPYING/`](https://github.com/Cisco-Talos/clamav/tree/main/COPYING) and list them all (and their checksums). 

You can append them like `LICENSE = "GPL-2.0-only & BSD-2-Clause & Unlicense"`.